corehq/apps/hqadmin/views/users.py

```diff
@@ -538,9 +538,15 @@ def get_timing_context(app, request_user_id):
         with app.timing_context:
             errors = app.validate_app()
             assert not errors, errors
-            app.make_build(
-                comment="Generated automatically during profiling",
-                user_id=request_user_id,
-                previous_version=app.get_latest_app(released_only=False),
-            )
+
+            with app.timing_context("build_zip"):
```

Comment:
What does adding a string to the timing context do?

Reply:
calling `with timing_context(string)` starts a new section in the results called `string`